src/docker-compose.yaml

```diff
@@ -0,0 +1,105 @@
+networks:
+    example-app-dev:
+        driver: bridge
+    traefik-proxy-blumilk-local:
+        external: true
+
+volumes:
+    example-app-postgres-data:
+        name: example-app-postgres-data
+    example-app-redis-data:
+        name: example-app-redis-data
+
+services:
+    app:
+        build:
+            context: ./environment/dev/app
+            dockerfile: Dockerfile
+            args:
+                - INSTALL_XDEBUG=${DOCKER_INSTALL_XDEBUG:-true}
+                - USER_ID=${DOCKER_HOST_USER_ID:-1000}
+        labels:
+            - "traefik.enable=true"
+            - "traefik.blumilk.environment=true"
+            # HTTP
+            - "traefik.http.routers.example-app-http-router.rule=Host(`example-app.blumilk.localhost`)"
+            - "traefik.http.routers.example-app-http-router.entrypoints=web"
+            # HTTP to HTTPS redirect
+            #      - "traefik.http.routers.example-app-http-router.middlewares=https-redirect@file"
+            # HTTPS
+            - "traefik.http.routers.example-app-https-router.rule=Host(`example-app.blumilk.localhost`)"
+            - "traefik.http.routers.example-app-https-router.entrypoints=websecure"
+            - "traefik.http.routers.example-app-https-router.tls=true"
+        container_name: example-app-app-dev
+        working_dir: /application
+        volumes:
+            - ./environment/dev/app/nginx.conf:/etc/nginx/nginx.conf:ro
+            - ./environment/dev/app/php.ini:/usr/local/etc/php/conf.d/zzz-overrides.ini:ro
+            - ./environment/dev/app/php-fpm.conf:/usr/local/etc/php-fpm.d/zzz-overrides.conf:ro
+            - ./environment/dev/app/supervisord.conf:/etc/supervisor/custom-supervisord.conf:ro
+            - .:/application
+        ports:
+            - ${DOCKER_APP_HOST_PORT:-3851}:80
+        networks:
+            - example-app-dev
+            - traefik-proxy-blumilk-local
+        restart: unless-stopped
+        depends_on:
+            database:
+                condition: service_healthy
+
+    database:
+        image: postgres:15.5-alpine3.18
```

Comment:
In tests there is `image: postgres:16.1-alpine3.18` so I think we should update it here too.